### DISCUSSION TITLE
Tips and Tricks for Discord and Slack

### DISCUSSION THREAD

**Discussion by Kernix13** [EddieHubCommunity/support#3403] [Ideas]
I'm new to the community, and to contributing, and I wanted to see if there was any interest in an advanced guide on using Discord and Slack. Or am I the only one who has difficulty finding messages in those collaboration tools? 

As an alternative, how about a doc on getting the most out of VS Code?

**Comment by eddiejaoude** [comment]
I don't use Slack anymore, I find it not collaborative like Discord.

Instead of a doc, what about a GitHub repo with a markdown file for any tips to share on VSCode? Then others could collaborate too

**Comment by Kernix13** [comment]
That's good to know about Slack. I sometimes see that mentioned in job descriptions. I'll stick to discord.

A repo for VSCode tips, tricks, and JSON settings would be good. I recently was shown the button to view Markdown files while in VSCode. Is that a repo that you would create here or should I create one and link to it? And just one md file or would we want different files for different languages, extensions, etc?

**Reply by eddiejaoude** [reply]
I recommend you create the repo and link to it.

Yes, start with 1 file, the `README.md` then late when it grows it can be broken out into smaller files

**Comment by Kernix13** [comment]
Okay, here is the link to my [VS Code Tips repo](https://github.com/Kernix13/vs-code-tips) with some shortcuts and a skeleton of different headings.
